internal/rbd/rbd_journal.go

```diff
@@ -25,6 +25,15 @@ import (
 	"github.com/ceph/ceph-csi/internal/util"
 )
 
+const (
+	// PVVolumeHandleAnnotationKey is the annotation key set on the PV object.
+	PVVolumeHandleAnnotationKey = "csi.ceph.io/volume-handle"
+	// PVReplicatedLabelKey is the label key set on PV object.
+	PVReplicatedLabelKey = "CephCSIReplicatedVolume"
```

Comment:
Same as with annotations, use `csi.ceph.io/repliacted-volume` to follow the naming other projects do

Reply:
done.